tests/db_migrator_input/config_db/reclaiming-buffer-warmreboot-input.json

```diff
@@ -0,0 +1,2138 @@
+    "BUFFER_QUEUE|Ethernet216|3-4": {
+        "profile": "[BUFFER_PROFILE|egress_lossless_profile]"
+    },
+    "BUFFER_QUEUE|Ethernet216|5-6": {
+        "profile": "[BUFFER_PROFILE|q_lossy_profile]"
+    },
+    "BUFFER_QUEUE|Ethernet224|0-2": {
+        "profile": "[BUFFER_PROFILE|q_lossy_profile]"
+    },
+    "BUFFER_QUEUE|Ethernet224|3-4": {
+        "profile": "[BUFFER_PROFILE|egress_lossless_profile]"
+    },
+    "BUFFER_QUEUE|Ethernet224|5-6": {
+        "profile": "[BUFFER_PROFILE|q_lossy_profile]"
+    },
+    "BUFFER_QUEUE|Ethernet232|0-2": {
+        "profile": "[BUFFER_PROFILE|q_lossy_profile]"
+    },
+    "BUFFER_QUEUE|Ethernet232|3-4": {
+        "profile": "[BUFFER_PROFILE|egress_lossless_profile]"
+    },
+    "BUFFER_QUEUE|Ethernet232|5-6": {
+        "profile": "[BUFFER_PROFILE|q_lossy_profile]"
+    },
+    "BUFFER_QUEUE|Ethernet240|0-2": {
+        "profile": "[BUFFER_PROFILE|q_lossy_profile]"
+    },
+    "BUFFER_QUEUE|Ethernet240|3-4": {
+        "profile": "[BUFFER_PROFILE|egress_lossless_profile]"
+    },
+    "BUFFER_QUEUE|Ethernet240|5-6": {
+        "profile": "[BUFFER_PROFILE|q_lossy_profile]"
+    },
+    "BUFFER_QUEUE|Ethernet248|0-2": {
+        "profile": "[BUFFER_PROFILE|q_lossy_profile]"
+    },
+    "BUFFER_QUEUE|Ethernet248|3-4": {
+        "profile": "[BUFFER_PROFILE|egress_lossless_profile]"
+    },
+    "BUFFER_QUEUE|Ethernet248|5-6": {
+        "profile": "[BUFFER_PROFILE|q_lossy_profile]"
+    },
+    "CABLE_LENGTH|AZURE": {
+        "Ethernet8": "300m",
+        "Ethernet184": "40m",
+        "Ethernet0": "300m",
+        "Ethernet248": "300m",
+        "Ethernet104": "300m",
+        "Ethernet240": "40m",
+        "Ethernet200": "40m",
+        "Ethernet168": "40m",
+        "Ethernet120": "300m",
+        "Ethernet144": "40m",
+        "Ethernet208": "40m",
+        "Ethernet160": "40m",
+        "Ethernet224": "40m",
+        "Ethernet56": "300m",
+        "Ethernet128": "40m",
+        "Ethernet72": "300m",
+        "Ethernet32": "300m",
+        "Ethernet16": "300m",
+        "Ethernet192": "40m",
+        "Ethernet96": "300m",
+        "Ethernet88": "300m",
+        "Ethernet80": "300m",
+        "Ethernet112": "300m",
+        "Ethernet152": "40m",
+        "Ethernet136": "40m",
+        "Ethernet48": "300m",
+        "Ethernet232": "40m",
+        "Ethernet216": "40m",
+        "Ethernet176": "40m",
+        "Ethernet40": "300m",
+        "Ethernet64": "300m",
+        "Ethernet24": "300m"
+    },
+    "DEVICE_METADATA|localhost": {
+        "hwsku": "ACS-MSN4700",
+        "default_bgp_status": "up",
+        "type": "LeafRouter",
+        "region": "None",
+        "hostname": "sonic",
+        "platform": "x86_64-mlnx_msn4700-r0",
+        "mac": "00:01:02:03:04:00",
+        "default_pfcwd_status": "disable",
+        "bgp_asn": "65100",
+        "deployment_id": "1",
+        "docker_routing_config_mode": "separated",
+        "cloudtype": "None"
+    },
+    "PORT|Ethernet0": {
```

Comment:
Don't see any admin down ports in this file

Reply:
Port Ethernet248 is. Sorry for putting it in the last...
The motivation is to verify whether the dynamic buffer profiles that were referenced by admin down ports only have been removed from APPL_DB after warm reboot.